src/registrar/utility/admin_helpers.py

```diff
@@ -0,0 +1,41 @@
+from registrar.models.domain_request import DomainRequest
+from waffle.decorators import flag_is_active
+from django.template.loader import get_template
+
+
+def get_all_action_needed_reason_emails(request, domain_request):
+    """Returns a dictionary of every action needed reason and its associated email
+    for this particular domain request."""
+
+    emails = {}
+    for action_needed_reason in domain_request.ActionNeededReasons:
+        # Map the action_needed_reason to its default email
+        emails[action_needed_reason.value] = get_action_needed_reason_default_email(
+            request, domain_request, action_needed_reason.value
+        )
+
+    return emails
+
+
+def get_action_needed_reason_default_email(request, domain_request, action_needed_reason):
+    """Returns the default email associated with the given action needed reason"""
+    if not action_needed_reason or action_needed_reason == DomainRequest.ActionNeededReasons.OTHER:
+        return None
+
+    if flag_is_active(request, "profile_feature"):  # type: ignore
+        recipient = domain_request.creator
+    else:
+        recipient = domain_request.submitter
+
```

Comment:
The profile feature flag is being removed in my ticket. Could we remove it here?